libs/utils/src/Hardware.ts

```diff
@@ -122,9 +122,9 @@ export class MemoryHardware implements Hardware {
     connectAccessibleController?: boolean
     connectCardReader?: boolean
   } = {}) {
-    this.setPrinterConnected(connectPrinter)
-    this.setAccessibleControllerConnected(connectAccessibleController)
-    this.setCardReaderConnected(connectCardReader)
+    void this.setPrinterConnected(connectPrinter)
+    void this.setAccessibleControllerConnected(connectAccessibleController)
+    void this.setCardReaderConnected(connectCardReader)
```

Comment:
This implies to me that we should just use an implicit constructor and make this a `static async` builder method.

Reply:
@carolinemodic I don't have an opinion on this but worth addressing before merging.